### ISSUE TITLE
Continuation of the Community Call Survey

### ISSUE THREAD

**Issue by natenatesoo** [CommonsBuild/coordination#1157]
## 👇🏽  What is this issue about? Give as much detail as you can. 
Continue to gather information that will help guide and improve the structure of our Community Call.

## ✨  What does success look like? (i.e., the criteria to meet to call this issue done)
1. Distribute survey for new members at each Community Call.
2. Continue to update the metrics and findings of the survey with Stewards.
3.

## 🧍🏻‍♀️🧍🏻‍♂️  Who should be part of the Advice Process? (People who should be consulted about this issue. Those who will have valuable input.)
@liviade1 

## 🔗  Are there links or docs related to this issue?
https://docs.google.com/forms/d/e/1FAIpQLSchdoaUQojbyoqSs-5CY6yMM6q4Rm8bhho2iiyh-EAW_m6aWg/viewform

## 👀  Who should be in the loop about this issue?
@tamarandom

**Comment by tamarandom** [comment]
Hey @natenatesoo should this still be open? Is there anything left to do?

**Comment by natenatesoo** [comment]
Yes.  This is ongoing.  I don't have access to the results, but they are being distributed post community-calls to all first-time attendees.

**Comment by liviade1** [comment]
I added Nate as an editor

**Comment by natenatesoo** [comment]
Link to Results: https://docs.google.com/forms/d/1E6E-G7LMRBjsfju_jbVnUfI5FSQp2EmvxxRF0Qkua58/edit#responses
